app/src/main/java/io/appium/uiautomator2/core/AxNodeInfoExtractor.java

```diff
@@ -32,34 +32,50 @@
 
 public abstract class AxNodeInfoExtractor {
 
+    // The timeout argument only affects the lookup timeout for UiObject instances
+    // null timeout value means to use the default
+    // `Configurator.getInstance().getWaitForSelectorTimeout()` timeout which equals to 10000ms
+
     @Nullable
     public static AccessibilityNodeInfo toNullableAxNodeInfo(Object object) {
```

Comment:
Does this repository still have this usage...?
Wondered all of them are `AccessibilityNodeInfo toNullableAxNodeInfo(Object object, @Nullable Long timeoutMs)`  by this change

Reply:
makes sense. Removed